packages/workspace/src/schematics/remove/lib/update-root-jest-config.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  SchematicContext,
+  SchematicsException,
+  Tree,
+} from '@angular-devkit/schematics';
+import { getWorkspace, insert, RemoveChange } from '@nrwl/workspace';
+import * as ts from 'typescript';
+import { getSourceNodes } from '@nrwl/workspace/src/utils/ast-utils';
+import { from, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+/**
+ * Updates the root jest config projects array and removes the project.
+ *
+ * @param schema The options provided to the schematic
+ */
+export function updateRootJestConfig(schema) {
+  return (tree: Tree, _context: SchematicContext): Observable<Tree> => {
+    return from(getWorkspace(tree)).pipe(
+      map((_) => {
+        const projectToRemove = schema.projectName;
+
+        if (!tree.exists('jest.config.js')) {
+          throw new SchematicsException('Root jest.config.js not found');
```

Comment:
Could we just exit instead of throwing? There might be workspace without the root config or where the root config is named differently.

Reply:
agree

Reply:
@vsavkin Of course. I will change this tonight

Reply:
@vsavkin Should a warning be displayed if the jest config file could not be found?